src/App.jsx

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
+
+import { Sum } from './components/Sum';
+
```

Comment:
remove unnecessary lines
```suggestion
import { Sum } from './components/Sum';
```